backend/apps/llm/services.py

```
@@ -94,28 +107,92 @@ def _set_environment_variables(self):
+        # === Provider 分发 ===
+        if self.is_mock:
+            logger.info(f"[{self.request_id}] 使用 Mock 模式进行代码审查")
+            return self._review_with_mock(mr_info, start_time)
+
+        if self.provider == 'claude':
+            logger.info(f"[{self.request_id}] 使用 Claude CLI 模式进行代码审查")
+            return self._review_with_claude_cli(
+                repo_path, mr_info, commit_range, custom_prompt, start_time
+            )
+
+        # 所有 API 模式 provider（unionllm, litellm, openai, deepseek, gemini）
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)